hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -1546,22 +1544,23 @@ def create_rcg(self, **kwargs):
                 try:
                     self.client.modifyRemoteCopyGroup(rcg_name, opt)
                 except Exception as ex:
-                    msg = (_("There was an error setting the sync period for "
-                             "the remote copy group: %s.") %
-                           six.text_type(ex))
+                    msg = "Error encountered while setting the sync period "\
+                          "for the remote copy group: %s" % six.text_type(ex)
                     LOG.error(msg)
                     raise exception.HPERemoteCopyGroupBackendAPIException(
                         data=msg)
 
+        import pdb
+        pdb.set_trace()
         try:
```

Comment:
is it safe to have this pdb call here ?

Reply:
Was debugging a unit test when I put this and forgot to remove from the code before check in. Removed it.